src/pages/index.js

```diff
@@ -0,0 +1,123 @@
+import Head from "next/head";
+import Modal from "@/components/Modal";
+import ProfilePic from "../../public/assets/profile-pic.jpg";
+import Skills from "@/components/Skills";
+import Projects from "@/components/Projects";
+import Footer from "@/components/Footer";
+import NavBar from "@/components/NavBar";
+import ScrollToTop from "react-scroll-to-top";
+
+//react icons
+import { GiMoonBats } from "react-icons/gi";
+import {
+  AiFillLinkedin,
+  AiFillGithub,
+  AiFillCodeSandboxCircle,
+} from "react-icons/ai";
+import { FaFreeCodeCamp } from "react-icons/fa";
+import { FaArrowUp } from "react-icons/fa";
+
+import React, { useState } from "react";
+
+//to include an img in next you have to import the next/image component
+import Image from "next/image";
+import Testimonials from "@/components/Testimonials";
+
+export default function Home() {
+  const [openModal, setOpenModal] = useState(false);
+
+  //dark theme
+  const [darkMode, setDarkMode] = useState(false);
+
+  return (
+    <div className={darkMode ? "dark" : ""}>
+      <Head>
+        <title>Octavian Busuioc Portofolio</title>
```

Comment:
Vezi că pe celelalte pagini nu ai mai pus titlu. Arată puțin neglijent. 

Pentru SEO, aș sugera să pui și descriere.